src/middleware/router.js

```diff
@@ -13,10 +13,19 @@ import * as utils from '../utils'
 import * as messageStore from '../middleware/messageStore'
 import * as events from '../middleware/events'
 import { promisify } from 'util'
+import { getGridFSBucket } from '../contentChunk'
+import { Writable, Readable } from 'stream';
+import util from 'util'
+import mongodb from 'mongodb'
+import { connectionDefault } from '../config'
+import { brotliCompressSync } from 'zlib';
 
 config.mongo = config.get('mongo')
 config.router = config.get('router')
 
+
```

Comment:
Remove this unnecessary space

Reply:
addressed

Reply:
commit fixing this - https://github.com/jembi/openhim-core-js/pull/1036/commits/1d490191fae6ec448f38737d313ce667923f18c4